src/components/Combobox.tsx

```diff
@@ -193,8 +194,8 @@ function Combobox<T>({
       <DropdownItem
         disabled={option.disabled}
         className={`${isOptionVisible(option) ? '' : 'sr-only'}`}
-        key={`${option.value}`}
-        id={`option-${option.value}`}
+        key={`${JSON.stringify(option.value)}`}
```

Comment:
We should only JSON.stringify() once.  Doing this multiple times will be expensive.